### PR TITLE
Sign up & login

### PR DESCRIPTION
**Pull request by iprankurpandey**

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <title>Agri Ecom</title>
+    <link rel="stylesheet" href="/login.css">
+</head>
+
+<body>
+
+    <div class="navigation-menu">
+        <div class="left-side-panel" title="menu-item">
```
Comment (sri-rishi) [inline login.html:16]: use nav tag for navigation

File: signup.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin: auto;
+    /* padding: auto; */
+    background-color: silver;
+    box-shadow: 0px 0px 10px rgb(0, 0, 0, 0.5);
+    text-align: center;
+    /* display: block; */
```
Comment (sri-rishi) [inline signup.css:26]: remove comments

**Review by sri-rishi — CHANGES_REQUESTED** [review]

File: login.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+@import 'https://agri-component.netlify.app/css/components.css';
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body,
+html {
+    height: 100%;
```
Comment (omkarpatke) [inline login.css:11]: why are you giving height here I don't understand

File: login.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    font-weight: bold;
+    color: #000;
+    margin-bottom: 1rem;
+}
+
+.search-panel {
+    display: flex;
+}
+
+.mtrlshop {
```
Comment (omkarpatke) [inline login.css:75]: I think this class name is not meaningful so make it more understandable it's a good practice.

**Review by omkarpatke — APPROVED** [review]
have some minor suggestions rest all look good.

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Pull request merged**